notebooks/Manage Map Areas with Group and Index.ipynb

```diff
@@ -0,0 +1,456 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Introduction\n",
+    "This notebook is designed to help ArcGIS Online / ArcGIS Enterprise administrators manage map areas. Depending on your needs, the manual map area tools in Field Maps Designer and item details may be adequate.  If your managing multiple map areas in multiple maps or having to accomodate frequent changes, a scripted approach may be preferred.  \n",
+    "\n",
+    "When should map areas will need to be recreated?\n",
+    "\n",
+    ">Re-creating the offline map area differs from updating the offline map area. When you use the Recreate action, it deletes all packages associated with the map area and re-creates them based on the offline map area's settings.\n",
+    ">\n",
+    ">The primary reason to re-create a map area is to pick up schema changes that have occurred after you created the offline map area. For example, if you add or delete a field or change an attribute value list or range (domains), you must re-create the offline map area to pick up those changes.\n",
+    ">\n",
+    ">Source: More info: https://doc.arcgis.com/en/arcgis-online/manage-data/take-maps-offline.htm\n",
+    "\n",
+    "This process assumes that you are the owner of web map and that you are using either Enterprise 11.4+ or ArcGIS Online. \n",
+    "\n",
+    "This script can be updated to run on older versions of Enterprise (pre 2.4 ArcGIS API for Python). You would need to update `from arcgis.map import Map` to `from arcgis.mapping import webmap` along with the where webmap is referenced in code. \n",
+    "More details on ArcGIS API for Python 2.4: https://www.esri.com/arcgis-blog/products/api-python/announcements/whats-new-in-arcgis-api-for-python-2-4-0-blog/\n",
+    "\n",
+    "\n",
+    "---\n",
+    "\n",
+    "### Setup \n",
+    "\n",
+    "The script utilizes three items that will ***need to be created*** and configured in your portal \n",
+    "\n",
+    "#### Hosted Feature Layer \n",
+    "##### `Manage Map Areas`\n",
+    "- Setup Notes: Download the zipped fgdb and add it as an item to your portal.  The hosted feature layer and table below are both included.\n",
+    "\n",
+    "    - File hosted on ArcGIS Online: https://www.arcgis.com/home/item.html?id=2ab1be89349e4e61be674fc3e3cbfcdf\n",
+    "\n",
+    "    ##### `Manage Map Area Index`\n",
+    "    - Purpose: The polygon hosted feature layer stores map area boundries and processing details.  You can specify if the map area applied to all web map in the group, or a specific web map item id\n",
```

Comment:
This is a really cool notebook Jake. Leveraging hosted feature layers and tables to execute map area creation in a group. Awesome!

Reply:
Thanks!